src/Views/WhatsNewPage.xaml.cs

```diff
@@ -21,6 +22,12 @@ namespace DevHome.Views;
 
 public sealed partial class WhatsNewPage : Page
 {
+    public Uri DevDrivePageKeyUri { get; private set; } = new ("ms-settings:disksandvolumes");

```

Comment:
Nit: Values do not change, can be converted to private readonly/const members.
```cs
    private readonly Uri _devDrivePageKeyUri = new ("ms-settings:disksandvolumes");
    private readonly Uri _devDriveLearnMoreLinkUri = new ("https://go.microsoft.com/fwlink/?linkid=2236041");
    private const string _devDriveLinkResourceKey = "WhatsNewPage_DevDriveCard/Link";
```

Reply:
nice updated